src/components/ReceiptImage.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {View} from 'react-native';
+import useThemeStyles from '@hooks/useThemeStyles';
+import EReceiptThumbnail from './EReceiptThumbnail';
+import Image from './Image';
+import ThumbnailImage from './ThumbnailImage';
+
+type Style = {height: number; borderRadius: number; margin: number};
+
+type ReceiptImageProps = {
+    transactionID?: string;
```

Comment:
also, not all of these are optional, right? We should make sure the types aren't marked as optional if they aren't